packages/kbn-optimizer/limits.yml

```diff
@@ -29,7 +29,7 @@ pageLoadAssetSize:
   dashboard: 82025
   dashboardEnhanced: 65646
   data: 454087
-  datasetQuality: 50624
+  datasetQuality: 123008
```

Comment:
Did we look into making these changes async?

Reply:
We introduced lazy loading for the components ending up into the following metrics

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app

| id | [before](https://github.com/elastic/kibana/commit/c6101f1e73e51d30438afc1e60b7e84bddec4ae8) | [after](https://github.com/elastic/kibana/commit/c5fc3a2d83a692b369cb147cb4175ff5773283bd) | diff |
| --- | --- | --- | --- |
| `datasetQuality` | 0.0B | 126.0KB | +126.0KB |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb

| id | [before](https://github.com/elastic/kibana/commit/c6101f1e73e51d30438afc1e60b7e84bddec4ae8) | [after](https://github.com/elastic/kibana/commit/c5fc3a2d83a692b369cb147cb4175ff5773283bd) | diff |
| --- | --- | --- | --- |
| `datasetQuality` | 40.2KB | 7.0KB | -33.2KB |